lldb-eval/value.cc

```diff
@@ -188,6 +189,8 @@ bool Value::IsFloat() { return type_.IsFloat(); }
 
 bool Value::IsPointer() { return type_.IsPointerType(); }
 
+bool Value::IsArray() { return type_.IsArrayType(); }
```

Comment:
This method is used only inside `Value`, so let's just inline it. Most of `Value::Is*` are here for convenience, because doing `value.type().Is*` is annoying.